manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -184,21 +191,22 @@ You can use ``isort`` to fix the order of imports:
 
 After you run the command above, you can fix what ``modernizer`` did not get right.
 
-3 Use Precompile
-----------------
+3 Use ``precompiler``
+---------------------
 
-You can make use of `plone.recipe.precompiler <https://github.com/plone/plone.recipe.precompiler>`_ to identify syntax errors quickly.
+You can make use of ``plone.recipe.precompiler`` to identify syntax errors quickly.
 This recipe compiles all Python code already at buildout-time, not at run-time.
 You will see right away when there is some illegal syntax.
 
```

Comment:
Same here, do we link to the package already somewhere else?

Reply:
See line 25, the first time it is mentioned. Easier to maintain one instance of hyperlinks than multiple, so go with the first.